external/fv3fit/tests/training/test_train_keras.py

```diff
@@ -1,149 +0,0 @@
-from typing import Iterable, Sequence
-import xarray as xr
-import pytest
-import logging
-import loaders
-import fv3fit
-import numpy as np
-import tempfile
-
-from fv3fit.keras._training import set_random_seed
-
-logger = logging.getLogger(__name__)
-
-
-@pytest.fixture(params=["DenseModel"])
-def model_type(request) -> str:
-    return request.param
-
-
-@pytest.fixture(params=["mse"])
-def loss(request) -> str:
-    return request.param
-
-
-@pytest.fixture(params=[{"width": 4, "depth": 3}])
-def hyperparameters(request, model_type, loss) -> dict:
-    if model_type == "DenseModel":
-        hyperparameters = request.param
-        if loss:
-            hyperparameters["loss"] = loss
-        return hyperparameters
-    else:
-        raise NotImplementedError(model_type)
-
-
-@pytest.fixture
-def model(train_config) -> fv3fit.Estimator:
-    return fv3fit.keras.get_model(
-        train_config.model_type,
-        loaders.SAMPLE_DIM_NAME,
-        train_config.input_variables,
-        train_config.output_variables,
-        train_config,
-    )
-
-
-def test_reproducibility(
```

Comment:
I know the keras reproducibility isn't on the same level as the sklearn tests added in this PR, where the same hash ought to be able to be reproduced, but I think we should keep a test that the keras model predictions stay within some tolerance as long as you are training with the same random seed.

Reply:
Actually while looking into this just now I realize that the original test was too lenient because the testing data coming from the tests/training/conftest was using zero arrays, which were trivial to match predictions for with different seeds... the test works as intended when it uses np.random.choice.

Reply:
Added a test for this.